color/src/colorspace.rs

```diff
@@ -525,6 +525,64 @@ impl ColorSpace for Rec2020 {
     }
 }
 
+#[expect(
+    clippy::doc_markdown,
+    reason = "these color space names should not be surrounded in backticks"
+)]
+/// 🌌 The ACEScg color space.
+///
+/// This color space is defined by the Academy Color Encoding System [specification][acescg].
+///
+/// The ACEScg color space is a linear color space. The wide gamut makes this color space useful as
+/// a working space for computer graphics.
+///
+/// Other ACES color spaces include ACES2065-1, ACEScc, and ACEScct. This crate currently does not
+/// support these color spaces.
+///
+/// The ACEScg components are `[R, G, B]`. The components are bounded to `[-65504.0, 65504.0]`,
+/// though it is unusual to clip in this color space.
+///
+/// The conversion between D65 and the ACES whitepoint is done with the standard Bradford linear
+/// chromatic adaptation transform.
+///
+/// [acescg]: https://docs.acescentral.com/specifications/acescg/
+#[derive(Clone, Copy, Debug)]
+pub struct AcesCg;
+
+impl ColorSpace for AcesCg {
+    const IS_LINEAR: bool = true;
+
+    const TAG: Option<ColorSpaceTag> = Some(ColorSpaceTag::AcesCg);
+
+    fn to_linear_srgb(src: [f32; 3]) -> [f32; 3] {
+        // XYZ_to_lin_sRGB * ACESwp_to_D65 * ACEScg_to_XYZ
```

Comment:
Should you use the exact rationals with the same `expect` thing that Rec2020 does?

Reply:
Tom used [Sage](https://www.sagemath.org/) to get those exact rational numbers. I don't know Sage, so it would take a fair bit more work for me to get those exact rationals. That work probably isn't worth it to "[guild the lily](https://github.com/linebender/color/pull/35#issuecomment-2474734716)" as Raph called it, since the resulting matrix should be the same.

I would guess the motivation for exact rationals is so someone could later calculate a more precise version of the matrix. As an alternative, providing documentation with enough information someone could derive the same result would serve the same purpose and would also be more informative.

Reply:
Marking as resolved, let me know if you disagree. I can add more documentation, but there's more documentation in general I'd like to add, which can be done in a later PR.